automation_tools/__init__.py

```diff
@@ -993,8 +990,7 @@ def setup_foreman_discovery(sat_version):
         run(package_install('foreman-discovery-image'))
 
     # Unlock the default Locked template for discovery
-    run('hammer -u admin -p {0} template update '
-        '--name "PXELinux global default" --locked "false"'
+    run('hammer -u admin -p {0} template update --name "PXELinux global default" --locked "false"'
```

Comment:
Use Hammer function from Hammer module (https://github.com/SatelliteQE/automation-tools/blob/master/automation_tools/satellite6/hammer.py#L72) instead ?

Reply:
Out of scope of this PR, I only unwrap the line